config/locales/en.yml

```diff
@@ -690,6 +694,14 @@ en:
         - the client is financially independent
         - the client is in local authority or foster care
       guidance: Guidance on means aggregation for under 18s
+    regular_income:
+      title: Does your client get regular income?
+      explainer: For example, payment from a full time job or local authority financial support for a care leaver. If they get regular income, they must have a full means test.
+      exception_intro: "Do not include:"
+      exceptions:
+        - part-time earnings (less than 16 hours per week)•  holiday jobs•  pocket money
```

Comment:
i think holiday jobs and pocket money need to be on their own lines

Reply:
Good spot, thanks! I've fixed.